doc/refman/Universes.tex

```diff
@@ -131,6 +131,52 @@
   polymorphically, not at a single instance.
 \end{itemize}
 
+\asection{{\tt Cumulative, NonCumulative}}
+\comindex{Cumulative}
+\comindex{NonCumulative}
+\optindex{Inductive Cumulativity}
+
+Inductive types, coinductive types, variants and records can be
+declared cumulative using the \texttt{Cumulative}. Alternatively,
+there is an option \texttt{Set Inductive Cumulativity} which when set,
```

Comment:
In the current state, if this command is present but not `Set Universe Polymorphism.` new inductive definitions are considered cumulative if and only if they are prefixed with `Polymorphic`. If this is the expected behavior, then this should be documented. Otherwise, this should be fixed.